### PR TITLE
Vulnerability fixes

### PR DESCRIPTION
**Pull request by mboneil10**
Hopefully this will fix it. Clicked through the app and nothing seems to have changed.

### CODE REVIEW

**Comment by mboneil10** [comment]
UGH CI failed.

**Comment by mboneil10** [comment]
```
found 21 vulnerabilities (11 low, 6 moderate, 4 high) in 8125 scanned packages
  21 vulnerabilities require semver-major dependency updates.
```
I'll try to figure this out.

File: ionic.config.json [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {
   "name": "PVTrAck",
-  "app_id": "",
-  "type": "ionic-angular",
   "integrations": {
     "cordova": {}
-  }
-}
+  },
+  "type": "ionic-angular"
```
Comment (Anbranin) [inline ionic.config.json]: What does putting it here do?

Comment (mboneil10) [reply ionic.config.json]: It was a fix that was made when I ran a command.

**Comment by mboneil10** [comment]
Hallelujah. Idk what happened for Travis to pass, but it's a miracle.

**Comment by Anbranin** [comment]
Computers are mysterious black boxes

**Review by Anbranin — APPROVED** [review]

**Pull request merged**